ha/util/health_generator/README.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+This python script can be used for feeding mock health events to HA.
+
+# Usage
+
+usage: health_event_publisher [options]
+
+Helps in publishing the mock health event
+
+positional arguments:
+  {publish,get-disks,get-cvgs,help}
+    publish             Publish the health message
+    get-disks           Displys the Disk Ids assosciated with the Node
+    get-cvgs            Displys the CVG Ids assosciated with the Node
+    help                see `help -h`
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -gdt, --get-data-nodes
+                        Get the list of data node ids
+  -gs, --get-server-nodes
+                        Get the list of server node ids
```

Comment:
- can we have a few samples of the full commands to be executed on command line  here ?

- Also restrictions/limitations should be added somewhere right ? ex This script needs to be executed from HA container only etc

Reply:
Added.